datashader/glyphs.py

```diff
@@ -73,6 +75,34 @@ def _compute_y_bounds_dask(self, df):
         return np.nanmin(ys), np.nanmax(ys)
 
 
+class _PolygonLike(_PointLike):
+    """_PointLike class, with Triangle-specific methods overriden.
+
+    Key differences from _PointLike:
+        - added self.z as a list, representing vertex weights
+        - constructor accepts additional kwargs:
+            * weight_type (bool): Whether the weights are on vertices (True) or on the triangles (False)
+            * interp (bool): Whether to interpolate (True), or to have one color per triangle (False)
```

Comment:
Is this class implementing polygons, or triangles?  The name indicates polygons, but all the docstrings indicate triangles. I've updated the docs to say that it's about shapes, but if that's not correct please rename and fix the documentation.